### PR TITLE
Artifacts on build

### LINKED ISSUE #30: Automatic build artifacts on github when tagging
we will support the linux platform, due to the jpeg library.

I found this guide that seems to do the trick:

https://medium.com/threefold-digital/creating-automated-multi-platform-golang-executable-binary-releases-with-xgo-and-github-actions-8877f7b72740

Move `internal/cli` to `cmd/rac`

### PR DESCRIPTION
**Pull request by joakimmoller**
A new github action, activated on tags starting with v (as in v0.0.1), builds two binaries one for linux and one for windows and adds them into the release artifacts.

Resolves #30

Added some instructions to the usage of 
```
$ rac
Extracts information from Innosat-MATS rac-files

Usage: rac [OPTIONS] rac-file ...
  -output string
        Directory to place images and/or timeseries data
  -skip-images
        Extract images from rac-files.
        (Default: false)
  -skip-timeseries
        Extract timeseries from rac-files.
        (Default: false)
  -stdout
        Output to standard out instead of to disk (only timeseries)
        (Default: false)

The tool can be used to scan rac files for contents use the -stdout flag and
use command line tools to scan for interesting information.
        rac -stdout my.rac  | grep STAT

Tip finding parsing errors:
        rac -stdout my.rac | grep -E -e".*Error:[^<:]+" -o

or if you want the Buffer conents which can be rather large if you are unlucky
        rac -stdout my.rac | grep -E -e".*Error:[^<]+" -o

2020/05/06 11:16:17 No rac-files supplied ```

### CODE REVIEW

**Review by skymandr — COMMENTED** [review]
Looks good. Have you checked on a Windows computer that the binary works?

**Comment by local-minimum** [comment]
Could you perhaps add the extended cli-example in the `./rac --help` for finding errors piping it to grep in #42 ?

**Comment by joakimmoller** [comment]
> Looks good. Have you checked on a Windows computer that the binary works?

YES it works! - but since the jpegdecode is not used yet I can't confirm that yet.

**Comment by joakimmoller** [comment]
> Could you perhaps add the extended cli-example in the `./rac --help` for finding errors piping it to grep in #42 ?

A bit of a scope creep, but OK - will do that.

File: cmd/rac/cli.go [hunk 1]
```diff
@@ -23,9 +23,18 @@ func myUsage() {
 	fmt.Println("Extracts information from Innosat-MATS rac-files")
 	fmt.Println()
 	fmt.Printf("Usage: %s [OPTIONS] rac-file ...\n", os.Args[0])
-	fmt.Println()
 	flag.PrintDefaults()
-	fmt.Println()
+	fmt.Println(`
+The tool can be used to scan rac files for contents use the -stdout flag and
```
Comment (skymandr) [inline cmd/rac/cli.go]: This sentence seems to be missing a period or a comma after `contents`. Perhaps change the period after `information` to `, e.g.:` as well? Should probably be a `:` after `unlucky` as well.

Comment (skymandr) [reply cmd/rac/cli.go]: (I think I would prefer `contents use` -> `contents. Use`.)

Comment (joakimmoller) [reply cmd/rac/cli.go]: Do you mind creating a suggestion (using the [+-] button) ?

Comment (skymandr) [reply cmd/rac/cli.go]: If I can find it...

Comment (skymandr) [inline cmd/rac/cli.go]:
```suggestion
The tool can be used to scan rac files for contents. Use the -stdout flag and
```

File: cmd/rac/cli.go [hunk 2]
```diff
@@ -23,9 +23,18 @@ func myUsage() {
...
+use command line tools to scan for interesting information.
```
Comment (skymandr) [inline cmd/rac/cli.go]:
```suggestion
use command line tools to scan for interesting information, e.g.:
```

File: cmd/rac/cli.go [hunk 3]
```diff
@@ -23,9 +23,18 @@ func myUsage() {
...
+	rac -stdout my.rac  | grep STAT
+
+Tip finding parsing errors:
+	rac -stdout my.rac | grep -E -e".*Error:[^<:]+" -o
+
+or if you want the Buffer conents which can be rather large if you are unlucky
```
Comment (skymandr) [inline cmd/rac/cli.go]:
```suggestion
or if you want the Buffer conents which can be rather large if you are unlucky:
```

File: cmd/rac/cli.go [hunk 4]
```diff
@@ -23,9 +23,18 @@ func myUsage() {
...
 	fmt.Printf("Usage: %s [OPTIONS] rac-file ...\n", os.Args[0])
-	fmt.Println()
 	flag.PrintDefaults()
-	fmt.Println()
+	fmt.Println(`
+The tool can be used to scan rac files for contents. Use the -stdout flag and
+use command line tools to scan for interesting information, e.g.:
+	rac -stdout my.rac  | grep STAT
+
+Tip finding parsing errors:
```
Comment (skymandr) [inline cmd/rac/cli.go]:
```suggestion
Tip for finding parsing errors:
```

File: cmd/rac/cli.go [hunk 5]
```diff
@@ -23,9 +23,18 @@ func myUsage() {
...
+	rac -stdout my.rac | grep -E -e".*Error:[^<:]+" -o
+
+or if you want the Buffer conents which can be rather large if you are unlucky:
```
Comment (skymandr) [inline cmd/rac/cli.go]:
```suggestion
or if you want the Buffer contents which can be rather large if you are unlucky:
```

**Review by skymandr — APPROVED** [review]
Two more language comments. Please commit those, then I'm happy.

**Pull request merged**